backend/src/main/java/com/bakdata/conquery/resources/admin/rest/UIProcessor.java

```diff
@@ -57,8 +57,9 @@ public class UIProcessor {
 	@Getter
 	private final AdminProcessor adminProcessor;
 
-	public DatasetRegistry getDatasetRegistry() {
-		return adminProcessor.getDatasetRegistry();
+	// todo(tm): UIContext requires a distributed context. Should it just be disabled if SQL is enabled?
```

Comment:
Das AdminUI ist auch im SQL modus relevant, was sind denn die Stellen an denen du Probleme hast? Vmtl musst du bestimmte komponenten weiter unten im Baum aufbrechen (vA Imports)

Reply:
Nur `UIContext`, der wiederum nur an den `UIView` übergeben wird, da aber nicht benutzt wird. Kann ich den löschen?

Reply:
Die 2 Methoden wurden tatsächlich nicht benutzt, aber das ganze ftl in der AdminUI ist natürlich auch sehr fest auf Shards etc gegossen. Das würde ich mir nächste Woche mal anschauen.

Bsp.: backend/src/main/resources/com/bakdata/conquery/resources/admin/ui/index.html.ftl